package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@gpn-prototypes/frontend-configs",
+  "version": "0.0.2",
+  "main": "index.js",
+  "author": "csssr",
```

думаю надо указать или вообще убрать автора, если можно 
```suggestion
  "author": "gpn-prototypes",
```